.github/workflows/build.yml

```diff
@@ -263,8 +263,8 @@ jobs:
           curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y --profile=minimal --default-toolchain ${{ env.TOOLCHAIN }}
       - name: Pin the syn and regex dependencies
         run: |
-          cd fuzz && cargo update -p regex --precise "1.9.6" && cargo update -p syn --precise "2.0.106"
-          cd write-seeds && cargo update -p regex --precise "1.9.6" && cargo update -p syn --precise "2.0.106"
+          cd fuzz && cargo update -p regex --precise "1.9.6" && cargo update -p syn --precise "2.0.106" && cargo update -p quote --precise "1.0.41"
+          cd write-seeds && cargo update -p regex --precise "1.9.6" && cargo update -p syn --precise "2.0.106" && cargo update -p quote --precise "1.0.41"
```

Comment:
I couldn't find any context for this change. It's not a backport, or is it? I am assuming it isn't necessary in main because of the MSRV bump.

Reply:
Indeed, main has a higher MSRV.